layouts/shortcodes/docs/languages/exporters/prometheus-setup.md

```diff
@@ -3,7 +3,7 @@
 To send your metric data to [Prometheus](https://prometheus.io/), you can either
 [enable Prometheus' OTLP Receiver](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver)
 and use the [OTLP exporter](#otlp) or you can use the
-[`PrometheusHttpServer`](https://javadoc.io/doc/io.opentelemetry/opentelemetry-exporter-prometheus/latest/io/opentelemetry/exporter/prometheus/PrometheusHttpServer.html),
+[`PrometheusHttpServer`](https://open-telemetry.github.io/opentelemetry-js/classes/_opentelemetry_exporter_prometheus.PrometheusExporter.html),
```

Comment:
This is in a shortcode, so it is applied all languages using that shortcode, meaning we need to inject the correct URL as a parameter to the short code

Reply:
That also crossed my mind... Hmmm. Should we remove the link entirely?

Reply:
not even the name is consistent across languages, so yes, maybe "Prometheus Exporter" without link and without the code-style

Reply:
Edited in https://github.com/open-telemetry/opentelemetry.io/pull/5038/commits/8e4ec8ce6e9bfd58348f77d8f8cb2f70149a2c88, have a look!